styles/globals.css

```diff
@@ -120,15 +120,6 @@ a.link {
   right: 4rem;
 }
 
-.registrationTitle:after {
-  content: '';
-  display: block;
-  margin: 0 auto;
-  width: 75%;
-  padding-top: 15px;
-  border-bottom: 2px solid black;
-}
-
```

Comment:
I think this should be in a CSS module local to either the registration question component or the register page.

Reply:
If it's because of `:after`, there's support for [`before` and `after` psuedo-elements](https://tailwindcss.com/docs/hover-focus-and-other-states#before-and-after) in tailwind. Maybe use that if it's not too ugly.

Reply:
I don't remember what this was used for lol I think this was from an old figma design or something